tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/LoginSteps.cs

```diff
@@ -41,8 +30,8 @@ public LoginSteps(ScenarioContext context)
                 options.AddArguments("--headless");
             }
             
-            driver = new ChromeDriver(Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory), options);
-            wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            _driver = new ChromeDriver(Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory), options);
```

Comment:
One suggestion here is to move the initialization of the IWebDriver out into a hook. This will remove the coupling of the ChromeDriver for this step class. This will allow the IWebDriver interface to be injected into the constructor of this step class.